<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileItem</class>
 <widget class="QWidget" name="FileItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pb_pic">
       <property name="minimumSize">
        <size>
         <width>56</width>
         <height>56</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>56</width>
         <height>56</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/images/folders.png</normaloff>:/images/folders.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>56</width>
         <height>56</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lb_filename">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_filesize">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lb_uploadProgress">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font:16px</string>
       </property>
       <property name="text">
        <string>上传进度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="bar_uploadProgress">
       <property name="minimumSize">
        <size>
         <width>141</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>141</width>
         <height>31</height>
        </size>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pb_send">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_recv">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
